pkg/apis/packet/types_controlplane.go

```diff
@@ -23,4 +23,14 @@ import (
 // ControlPlaneConfig contains configuration settings for the control plane.
 type ControlPlaneConfig struct {
 	metav1.TypeMeta
+
+	// Persistence contains configuration for storage in the cluster.
+	// +optional
```

Comment:
```suggestion
```

tags are not specified in the internal API

Reply:
OK, changed